### PR TITLE
Message bug

### PR DESCRIPTION
**Pull request by mnzit**

### CODE REVIEW

File: messages/src/main/java/life/genny/messages/process/MessageProcessor.java [hunk 1]
```diff
@@ -126,26 +134,46 @@ private Object processMessageHelper(QMessageGennyMSG message) {
...
+                        log.error("Error fetching PRI_DEFAULT_MSG_TYPE");
+                        log.error("Exception: "+ ex.getMessage());
+                    }
+
+                    if(typeList != null){
+                        try {
+                            messageTypeList = typeList.stream().map(QBaseMSGMessageType::valueOf).toList();
+                        } catch (Exception e) {
+                            log.error(e.getLocalizedMessage());
+                            return CommonUtils.logAndReturn(log::error, e);
```
Comment (Expressionless) [inline messages/src/main/java/life/genny/messages/process/MessageProcessor.java]: Be careful about returning exception object. Is there a reason this method's return type is Object?

Comment (jasperRob) [reply messages/src/main/java/life/genny/messages/process/MessageProcessor.java]: Agreed. It is better to target specific exception types.

Comment (mnzit) [reply messages/src/main/java/life/genny/messages/process/MessageProcessor.java]: done

**Review by jasperRob — APPROVED** [review]

**Review by Expressionless — APPROVED** [review]
lgtm

**Pull request merged**